src/main/java/uk/gov/hmcts/cmc/claimstore/events/offer/AgreementCounterSignedCitizenActionsHandler.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.cmc.claimstore.events.offer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.AgreementCounterSigned;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.offers.MadeBy;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.COUNTER_SIGNING_PARTY;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.FRONTEND_BASE_URL;
+import static uk.gov.hmcts.cmc.domain.models.offers.MadeBy.CLAIMANT;
+import static uk.gov.hmcts.cmc.domain.models.offers.MadeBy.DEFENDANT;
+
+@Component
+public class AgreementCounterSignedCitizenActionsHandler {
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    @Autowired
+    public AgreementCounterSignedCitizenActionsHandler(
+        NotificationService notificationService,
+        NotificationsProperties notificationsProperties
+    ) {
+        this.notificationService = notificationService;
+        this.notificationsProperties = notificationsProperties;
+    }
+
+    @EventListener
+    public void sendNotificationToOtherParty(AgreementCountersignedEvent event) {
+        Claim claim = event.getClaim();
+        String targetEmail = null;
```

Comment:
With current implementation it is not needed to initiate variables. IntelliJ proposes to remove it.

REC - no blocker

If you decide to address it please do it in other 3 places in the file.